content/en/glossary/terms/data_source.md

```diff
@@ -0,0 +1,6 @@
+---
+title: data source
```

Comment:
This might be confusing, we also used data source to describe where the data for graphs are coming from, and that's a fixed product list. This might be broad and generic enough to leave out of the Glossary, but let me know what you think.